cli/podman.adoc

```diff
@@ -148,117 +191,147 @@ After you have linked to a network, you can check the link status:
 
+The `skupper` CLI has two options for exposing services that already exist on a host:
 
-NOTE: Services exposed on Kubernetes are not automatically available to Podman sites.
-This is the equivalent to Kubernetes sites created using `skupper init --enable-service-sync false`.
+* `expose` supports simple use cases, for example, a deployment with a single service.
+See xref:exposing-simple-services[] for instructions.
 
-.Example 01: Consuming a service from a Podman site
+* `service create` and `service bind` is a more flexible method of exposing services, for example, if you have multiple services for a deployment.
+See xref:exposing-complex-services[] for instructions.
 
-In this variation of the link:https://github.com/skupperproject/skupper-example-hello-world[hello world] example, the `backend` service is exposed on Kubernetes site and a Podman site is linked.
-You deploy the `frontend` as a container and that container can access the `backend` service.
+// Type: procedure
+[id="exposing-simple-services"] 
+=== Exposing simple services on the {service-network}
+This section describes how services can be enabled for a {service-network} for simple use cases.
 
-. Create a Podman site and link it to a Kubernetes site.
+.Prerequisites 
 
-. Check the service from the Podman site:
-+
-[source, bash]
-----
-$ skupper service status
+* A Skupper Podman site
 
-No services defined
-----
-+
-This result is expected because services exposed on Kubernetes are not automatically available to Podman sites.
+.Procedure
 
-. Create a service on the Podman site matching the service exposed on the Kubernetes site:
+. Run a server, for example:
 +
 [source, bash]
 ----
-$ skupper service create backend 8080
+$ podman run --name backend-target --network skupper --detach --rm -p 8080:8080 quay.io/skupper/hello-world-backend
 ----
++
+This step is not Skupper-specific, that is, this process is unchanged from standard processes for your host.
+
 
-. Validate the service from the Podman site by checking the backend API health URL:
+. Create a service that can communicate on the {service-network}:
 +
 --
 [source, bash]
 ----
-$ podman run -it --rm --network=skupper --name=myubi ubi8/ubi curl backend:8080/api/health
-
-OK
+$ skupper expose [host <hostname|ip>]
 ----
 
-This command runs a container using the `skupper` network and returns the results from `http://backend:8080/api/health`
---
+where 
 
-. Run the frontend as a container:
-+
-[source, bash]
+* `<host>` is the name of the host where the server is running. 
+For example, the name of the container if you run the server as a container.
+* `<ip>` is the IP address where the server is running
+
+For the example deployment in step 1, you create a service using the following command:
 ----
-$ podman run -dp 8080:8080 --name hello-world-frontend --network skupper quay.io/skupper/hello-world-frontend
+$ skupper expose host hello-world-backend --address backend --port 8080
```

Comment:
The host name here seems incorrect as in the `podman run` above you're using `--name backend-target`.